src/test/java/com/blackducksoftware/integration/hub/alert/accumulator/AccumulatorReaderTestIT.java

```diff
@@ -0,0 +1,24 @@
+package com.blackducksoftware.integration.hub.alert.accumulator;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.springframework.batch.item.NonTransientResourceException;
+import org.springframework.batch.item.ParseException;
+import org.springframework.batch.item.UnexpectedInputException;
+
+import com.blackducksoftware.integration.hub.alert.TestGlobalProperties;
+import com.blackducksoftware.integration.hub.dataservice.notification.NotificationResults;
+
+public class AccumulatorReaderTestIT {
+
+    // TODO We will need an upgraded hub instance to test this as it will make hub calls using API key
```

Comment:
Let's keep this uncommented. I'd rather have test failures for a few minutes per day (and know why) than miss a test and forget to check something.